onBoardProject/screens/login_screen.tsx

```diff
@@ -0,0 +1,87 @@
+import React, {Component, useState} from 'react';
+import {
+  SafeAreaView,
+  ScrollView,
+  StatusBar,
+  StyleSheet,
+  Text,
+  TextInput,
+  useColorScheme,
+  View,
+  Pressable,
+} from 'react-native';
+
+type MyProps = {
+  title: string;
+  email: string;
+  password: string;
+};
+
+export const LoginScreen: React.FC<{
+  title: string;
+}> = ({title}) => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  return (
+    <View style={styles.loginView}>
+      <Text style={styles.titleStyle}> {title} </Text>
+      <Text>Email</Text>
+      <TextInput
+        style={styles.input}
+        placeholder="name@domain.com"
+        onChangeText={val => setEmail(val)}
```

Comment:
Vc pode usar só a referência da função quando os parâmetros são iguais os q vc vai mandar: 

```suggestion
        onChangeText={setEmail}
```